src/style.css

```diff
@@ -1 +1,100 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: bold;
+  font-style: normal;
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+.search-bar__input {
+  display: block;
+  width: 100%;
+  margin-top: 20px;
```

Comment:
You have defined a top margin for the `.search-bar__input` class. As per the checklist, you should be consistent with your margins. Try to add either top or bottom margins, but not both to avoid potential margin collapse.